src/librbd/cache/pwl/ImageCacheState.h

```diff
@@ -37,7 +38,10 @@ class ImageCacheState {
   ~ImageCacheState() {}
 
   ImageCacheType get_image_cache_type() const {
-    return IMAGE_CACHE_TYPE_RWL;
+    if (cache_type == "rwl")
+      return IMAGE_CACHE_TYPE_RWL;
+
+    return IMAGE_CACHE_TYPE_SSD;
```

Comment:
Nit: for future-proofing, don't default to `SSD` but use an `UNKNOWN` enum

Reply:
Done